### PR TITLE
swagger added to report-processing-service

### PR DESCRIPTION
**Pull request by banadiga**

### CODE REVIEW

File: report-processing-service/package.json [hunk 1]
```diff
@@ -33,10 +36,14 @@
   },
   "dependencies": {
     "amqplib": "^0.5.2",
+    "fs": "0.0.1-security",
```
Comment (mpelekh) [inline report-processing-service/package.json]: what's the reason for this package?
https://www.npmjs.com/package/fs

there is native module `fs` to work with the file system

File: report-processing-service/package.json [hunk 2]
```diff
@@ -22,6 +24,7 @@
     "@types/koa-router": "^7.0.40",
     "@types/node": "^10.12.0",
     "@types/request-promise": "^4.1.42",
+    "find-free-port": "^2.0.0",
```
Comment (tmorozov) [inline report-processing-service/package.json]: I see no use of it in current PR

File: report-processing-service/src/controllers/swagger.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import {Context} from 'koa';
+import SwaggerService from '../services/swagger';
+
+@Route('/swagger.json')
+export class SwaggerController extends Controller {
+
+  private swaggerService = new SwaggerService();
+
+  @Get()
+  public async get(): Promise<any> {
```
Comment (tmorozov) [inline report-processing-service/src/controllers/swagger.ts]: don't we have a type for swagger.documentation?

File: report-processing-service/src/server.ts [hunk 4]
```diff
@@ -14,10 +14,10 @@ const reportReadyHandler = new ReportReadyHandler(reportClient, inspectorClient,
...
 
 export default consumer.init()
-    .then(() => {
-        const queueName = config.reportReadyForProcessingQueue.queueName;
-        return consumer.consumeMessagesFromQueue(queueName, reportReadyHandler.handler);
-    })
-    .then(() => {
-        return app;
-    });
+  .then(() => {
```
Comment (tmorozov) [inline report-processing-service/src/server.ts]: can't we use async/await here?

**Comment by tmorozov** [comment]
use of swagger service, looks overcomplicated for me, how about swagger2-koa?

**Pull request merged**